src/components/Organisms/EmailSignUp/_EmailSignUp.js

```diff
@@ -24,6 +24,7 @@ const EmailSignUp = ({
   privacyCopy,
   backgroundColour = 'deep_violet_dark',
   buttonColour = 'red',
+  buttonText,
```

Comment:
Probs makes the most sense to set the 'Subscribe' default here, rather than below, and pass _that_ directly to the button?

Reply:
For sure. Nice.